Changelog.md

```diff
@@ -1,3 +1,8 @@
+# 1.1.0 - 2016-04-28
+
+- Upgrade Minim to 0.14.0 and thus remove the short hand notation in fury
```

Comment:
I'm not sure, though if we are going by SemVer, this is a breaking change, isn't it?.